tiles-request-api/pom.xml

```diff
@@ -31,10 +31,21 @@ under the License.
     <version>1.0.7-SNAPSHOT</version>
     <name>Tiles request - API</name>
     <description>API for the Tiles Request framework.</description>
+
+    <properties>
+        <osgi.version>6.0.0</osgi.version>
+        <exam.version>4.11.0</exam.version>
+        <url.version>2.5.2</url.version>
+        <felix.version>5.6.4</felix.version>
+        <slf4j-api.version>1.7.25</slf4j-api.version>
+        <logback.version>1.2.2</logback.version>
```

Comment:
Trivial:
 We don't specify dependency versions by maven properties in the tiles projects.
 Please put these in with the dependency elements.